Bottom: f4f063222df46a4ff4017ef333880c4339e765f0
Top:    5ab27b7dc926cedddba90e96aafb800e28e039e5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-02 17:56:57 +0100

Build with asserts for polygons


---

diff --git a/src/polygon.c b/src/polygon.c
index b52e1f7..5e51b4c 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -73,6 +73,7 @@ dicer output is used for HIDs which cannot render things with holes
 /* special polygon editing routines
  */
 
+#undef NDEBUG
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
diff --git a/src/polygon1.c b/src/polygon1.c
index 4a761d5..03b02cd 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,6 +39,7 @@
       are marked
 */
 
+#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
@@ -105,7 +106,7 @@ if (UNLIKELY (((ptr) = malloc(sizeof(type))) == NULL)) \
 #undef DEBUG_JUMP
 #undef DEBUG_GATHER
 #undef DEBUG_ANGLE
-#undef DEBUG
+#define DEBUG
 #ifdef DEBUG
 #define DEBUGP(...) fprintf(stderr, ## __VA_ARGS__)
 #else
